src/fprime_gds/flask/updown.py

```diff
@@ -112,6 +113,53 @@ def post(self):
                 failed.append(key)
         return {"successful": successful, "failed": failed}
 
+    def save(self, file_storage: FileStorage):
+        """
+        This saves a `werkzeug.FileStorage` into this upload set.
+
+        :param file_storage: The uploaded file to save.
+        """
+        if not isinstance(file_storage, FileStorage):
+            raise TypeError("file_storage must be a werkzeug.FileStorage")
+
+        filename = Path(secure_filename(file_storage.filename)).name
+        dest_dir = Path(self.dest_dir)
+
+        # not necessary
```

Comment:
Add permissions check and is necessary, to keep independent of StandardPipeline

Reply:
Fixed